reanalysis/interpretation_runner.py

```diff
@@ -415,18 +333,30 @@ def main(
         vep_ht_tmp = output_path(
             'vep_annotations.ht', get_config()['buckets'].get('tmp_suffix')
         )
-        annotation_jobs = annotate_vcf(input_path, batch=batch, vep_out=vep_ht_tmp)
+        # generate the jobs which run VEP & collect the results
+        vep_jobs = add_vep_jobs(
+            b=get_batch(),
+            vcf_path=to_path(input_path),
+            tmp_prefix=to_path(output_path('vep_temp', 'tmp')),
+            scatter_count=get_config()['workflow'].get('scatter_count', 50),
+            out_path=to_path(vep_ht_tmp),
+        )
 
         # if convert-to-VCF job exists, assign as an annotation dependency
         if prior_job:
-            for job in annotation_jobs:
+            for job in vep_jobs:
                 job.depends_on(prior_job)
 
-        # apply annotations
-        prior_job = annotated_mt_from_ht_and_vcf(
-            input_vcf=input_path, batch=batch, job_attrs={}, vep_ht=vep_ht_tmp
+        # Apply the HT of annotations to the VCF, save as MT
+        anno_job = annotate_cohort_jobs(
+            b=get_batch(),
+            vcf_path=to_path(input_path),
+            vep_ht_path=to_path(vep_ht_tmp),
+            out_mt_path=to_path(ANNOTATED_MT),
+            checkpoint_prefix=to_path(output_path('annotation_temp', 'tmp')),
+            depends_on=vep_jobs,
         )
-        prior_job.depends_on(*annotation_jobs)
+        prior_job = anno_job[-1]
```

Comment:
```suggestion
        prior_job = anno_job[-1]
        prior_job.depends_on(*vep_jobs)
```

the OG annotate_cohort_jobs doesn't implement `depends_on`, so this needs to be set separately

Reply:
What do you mean? `annotate_cohort_jobs` would be updated in `cpg-workflows`. `depends_on` is needed for the dataproc version of annotation

Reply:
Ahhhhh, I didn't realise this introduces dataproc as a dependency again - that will be a problem for working with MSFT

Reply:
That's optional, there is `use_dataproc` flag. I just find it easier to debug with dataproc

Reply:
Updated to make sure use_dataproc=False